public/app/features/alerting/unified/components/rule-list/EvaluationGroup.tsx

```diff
@@ -0,0 +1,117 @@
+import { css, cx } from '@emotion/css';
+import React, { PropsWithChildren, ReactNode } from 'react';
+
+import { GrafanaTheme2 } from '@grafana/data';
+import { useStyles2, Badge, Button, Dropdown, Menu, Stack, Text, Icon } from '@grafana/ui';
+
+import { MetaText } from '../MetaText';
+import { Spacer } from '../Spacer';
+
+interface EvaluationGroupProps extends PropsWithChildren {
+  name: string;
+  description?: ReactNode;
```

Comment:
what is this description ? I don't see that is being used

Reply:
I think I initially used it for showing the evaluation interval, which I've moved in to its own prop since then and probably didn't clean this one up :D 

I can't think of anything other then "contains <number> rules" or something that we could use it for (which is going to be difficult with pagination) so I'll just remove it for now.